includes/tracks/class-wc-tracks.php

```diff
@@ -17,6 +17,18 @@ class WC_Tracks {
 	 */
 	const PREFIX = 'wca_test_';
 
+	/**
+	 * Option name for total store revenue.
+	 */
+	const REVENUE_CACHE_OPTION = 'woocommerce_tracker_orders_gross';
```

Comment:
For what I can see for `WC_Tracker::get_order_totals` we may in the future save more data, so why not already change it to `woocommerce_tracker_orders_totals`?